django_lightweight_queue/app_settings.py

```diff
@@ -21,3 +21,7 @@ def setting(suffix, default):
 REDIS_HOST = setting('REDIS_HOST', '127.0.0.1')
 REDIS_PORT = setting('REDIS_PORT', 6379)
 REDIS_PREFIX = setting('REDIS_PREFIX', '')
+
+ENABLE_PROMETHEUS = setting('ENABLE_PROMETHEUS', True)
```

Comment:
The idea of backwards-compatible here is sketchy to me -- this is adding quite a bunch more stuff which other consumers may not need (and may not be expecting).

Note also that this ends up binding on a publicly accessible port on the hosts. This is a security choice which the deploying engineer should IMO actively choose do use, rather than have it foisted upon them by default.